EasyPost.Tests/ServicesTests/AddressServiceTest.cs

```diff
@@ -91,9 +88,8 @@ public async Task TestCreateVerifyStrictArray()
             UseVCR("create_verify_strict_array");
 
             Dictionary<string, object> addressData = Fixtures.CaAddress1;
-            addressData.Add("verify_strict", new List<bool> { true });
 
-            Address address = await Client.Address.Create(addressData);
+            Address address = await Client.Address.Create(addressData, verifyStrict: true);
```

Comment:
Please rollback these changes. As with other items, we were going to focus on updated params at a later date across all libs. Let's not do that now

Reply:
See https://github.com/EasyPost/easypost-csharp/pull/360#discussion_r1003570705

Reply:
Replied in https://github.com/EasyPost/easypost-csharp/pull/360#discussion_r1003578479

Reply:
Resolved